gui/packages/ubuntupro/test/pages/landscape/landscape_page_test.dart

```diff
@@ -245,8 +267,30 @@ void main() {
       await tester.enterText(fqdnInput, '::');
       await tester.pump();
 
-      final errorText = find.text(lang.landscapeFQDNError);
-      expect(errorText, findsOne);
+      final fqdnErrorText = find.text(lang.landscapeFQDNError);
+      expect(fqdnErrorText, findsOne);
+
+      final chooseFileButton = find.ancestor(
+        of: find.text(lang.landscapeFilePicker),
+        matching: find.byType(FilledButton),
+      );
```

Comment:
Unless the button semantics are strictly important for the test case, I prefer to find buttons my test will tap just by `find.text()` (or `find.button()` from `yaru_test`). It's harder to miss a change in the localizable text getter than a change in the button type, which may lead to a more likely test failure in the future.